person.rb

```diff
@@ -49,18 +50,13 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals << rental
+  end
```

Comment:
- [ ] The person itself should be sent as a parameter to create the new `rental` achieving the 'has-many' association
### Sample code:
```
def add_rental(book, date)
    Rental.new(date, book, self)
 end
```